### ISSUE TITLE
QR Code reader give better feedback for sucessful scan.

### ISSUE THREAD

**Issue by Artim96** [thunderbird/thunderbird-android#8465]
### Checklist

- [X] I have used the search function to see if someone else has already submitted the same feature request.
- [X] I will describe the problem with as much detail as possible.
- [X] This issue only contains a request for one single feature, **not** multiple (related) features.

### App version

8.0

### Problem you are trying to solve

When trying to transfer settings from Thunderbird for Desktop, you have to scan at least one QR Code. Especially when you have to scan multiple QR Codes, the feedback for successful scanning is very lacking. Other scanners give you much better visual feedback or even haptic feedback. Instead, only the little text below the viewer indicates success.

### Suggested solution

Give much more clear visual and/or haptic feedback upon successful scan.

**Comment by kewisch** [comment]
Agreed, we talked about this prior to launch but didn't have time to work out better feedback. The mockups have more elaborate styling with success and error case, and haptic feedback would be great as well.

**Comment by Kurash-Rex** [comment]
[See Issue #10347](https://github.com/thunderbird/thunderbird-android/issues/10347#issuecomment-5106891424). I recently imported eight email accounts via three QR codes. It is not evident via messaging of haptic feedback that Thunderbird for Andriod is ready to scan the next QR code.
